apps/web/src/server/appeals/appeal-details/appeal-details.router.js

```diff
@@ -24,4 +24,19 @@ router.route('/:appealId/enter-start-date').get(async (request, response) => {
 	}
 });
 
+router.route('/:appealId/start-date-entered').get(async (request, response) => {
+	const appealDetails = await appealDetailsService
+		.getAppealDetailsFromId(request.params.appealId)
+		.catch((error) => logger.error(error));
+
+	if (appealDetails) {
+		response.render('appeals/appeal/start-date-entered.njk', {
+			appeal: {
+				id: appealDetails?.appealId,
+				reference: appealDetails?.appealReference
+			}
+		});
+	}
+});
```

Comment:
It makes sense to have a conditional to render the page if `appealDetails` is set, but should we have a fallback render/redirect after the conditional to catch the scenario where it's not set?

What would the user see currently, perhaps a white screen?

I see in the description that it's a temporary router so perhaps this will be covered later.

Reply:
It's a good point. I think ideally we should have an error page designed for this scenario (even if it's a generic one across the service). As you say though, this is a temporary router, just to enable the views to be worked on, so this should be addressed as part of BOAT-103/104 (router/controller tickets)

Reply:
:+1: